packages/terra-clinical-result-view/tests/wdio/result-view-spec.js

```diff
@@ -0,0 +1,46 @@
+/* global browser, Terra */
```

Comment:
We should also be testing if the themeable variables are applied.

Reply:
Added tests ( and addresses other comments)